scheduler/scheduler.yml

```diff
@@ -41,13 +41,6 @@
   ansible.builtin.import_playbook: ../utils/update_user_repo.yml
   when: not ( hostvars['127.0.0.1']['update_user_repo_executed'] | default(false) | bool )
 
-- name: Apply common Slurm installation and config
-  hosts: slurm_control_node, slurm_node, login
-  gather_facts: false
```

Comment:
did we handle all the tasks in slurm_common role? can we have the existing roles deleted in this PR if added